alf/trainers/policy_trainer.py

```diff
@@ -213,7 +213,15 @@ def _restore_checkpoint(self):
             algorithm=self._algorithm,
             metrics=nn.ModuleList(self._algorithm.get_metrics()))
 
-        recovered_global_step = checkpointer.load()
+        try:
```

Comment:
Do we still have  the original error message about which parameter is the problem? It is still useful for debugging in some cases.

Reply:
Good point, Now also appended the original error message in the new commit. 
So now we have both the high-level guidance and low-level details.